dashboard/test/ui/runner.rb

```diff
@@ -25,6 +27,26 @@
 
 ENV['BUILD'] = `git rev-parse --short HEAD`
 
+S3_LOGS_BUCKET = 'cucumber-logs'
+S3_LOGS_PREFIX = GitUtils.current_branch
```

Comment:
Realized just after sending this: my main concern is that we don't pollute the logs for the "test" branch with local runs. So, maybe this is not a problem, unless developers frequently check out a branch locally named "test", which seems unlikely.